msi/release.sh

```diff
@@ -177,6 +177,9 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
             <Component Directory="INSTALLFOLDER:\\usr\\share\\git\\">
                 <File Id="ReleaseNotes_Css" Source="$BUILD_EXTRA_WINPATH\\usr\\share\\git\\ReleaseNotes.css" />
             </Component>
+            <Component Directory="INSTALLFOLDER">
+                <RegistryValue Root="HKMU" Key="Software\GitForWindows" Name="CurrentVersion" Type="string" Value="\$(var.Version)" />

```

Comment:
A very good question. Call it "heat of the moment". I'll look into it, but I suspect you're right that this could line in `GitProduct.wxs`.